src/plone/restapi/serializer/user.py

```diff
@@ -57,13 +58,29 @@ def __call__(self):
         return data
 
 
-@implementer(ISerializeToJson)
+@implementer(ISerializeToJsonSummary)
 @adapter(IMemberData, IRequest)
-class SerializeUserToJson(BaseSerializer):
-    pass
+class SerializeUserToJsonSummary(BaseSerializer):
+    def __call__(self):
+        data = super().__call__()
+        return data
 
 
-@implementer(ISerializeToJsonSummary)
+@implementer(ISerializeToJson)
 @adapter(IMemberData, IRequest)
-class SerializeUserToJsonSummary(BaseSerializer):
-    pass
+class SerializeUserToJson(BaseSerializer):
+    def __call__(self):
+        data = super().__call__()
+        user = self.context
+        groups = user.getGroups()
+        batch = HypermediaBatch(self.request, groups)
```

Comment:
@ksuess Is it possible to rebase your PR on top of https://github.com/plone/plone.restapi/pull/1325
I see there're a few duplicates here: https://github.com/plone/plone.restapi/pull/1325/files#diff-6e6f94528c351affcc4311e1acb6eecaae4a33e6213fbad203a23886e5d4556a

Reply:
@nileshgulia1 It was meant as a cleaned up replacement of yours for the needs of https://github.com/plone/volto/issues/3018 Especially because I thought your work is on wait. But sure, I can rebase to yours.